common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -172,4 +180,55 @@ public class CommonValue {
             + USER_FIELD_MAPPING
             + "    }\n"
             + "}";
+
+    public static final String ML_CONNECTOR_INDEX_MAPPING = "{\n"
+            + "    \"_meta\": {\"schema_version\": "
+            + ML_CONNECTOR_SCHEMA_VERSION
+            + "},\n"
+            + "    \"properties\": {\n"
+            + "      \""
+            + DetachedConnector.CONNECTOR_NAME_FIELD
+            + "\" : {\"type\":\"text\",\"fields\":{\"keyword\":{\"type\":\"keyword\",\"ignore_above\":256}}},\n"
+            + "      \""
+            + DetachedConnector.CONNECTOR_VERSION_FIELD
+            + "\" : {\"type\": \"keyword\"},\n"
+            + "      \""
+            + DetachedConnector.CONNECTOR_DESCRIPTION_FIELD
+            + "\" : {\"type\": \"text\"},\n"
+            + "      \""
+            + DetachedConnector.CONNECTOR_PROTOCOL_FIELD
```

Comment:
I think protocol should be keyword, like `http` right?

Reply:
Yes, updated already.